models/issues/issue_search.go

```diff
@@ -50,8 +50,9 @@ type IssuesOptions struct { //nolint
 	PriorityRepoID int64
 	IsArchived     optional.Option[bool]
 	Org            *organization.Organization // issues permission scope
+	Owner          *user_model.User           // issues permission scope
```

Comment:
Why it should duplicate with "Org"?

Reply:
b63e059c5f meged Org and Owner as `Owner`.